### DISCUSSION TITLE
3D Mace as 2D in the inventory

### DISCUSSION THREAD

**Discussion by Leooo1702** [BedrockTweaks/Files#195] [Ideas]
The Trident is in 2D in your inventory and in 3D when in your hand, so it would make sense for the Mace to also be 2D in your inventory and 3D in your hand

**Comment by SquatchHunter** [comment]
3d mace exists now, however, due to how minecraft has done its icon strategy for items like the mace and our "lack of access" currently, it will be 3d or 2d in both locations - we cannot do a 3d hand held item without making the icon 3d **FOR NOW**, the plan is once we have access to vanilla items and can override them we should be able to swap the icon & "attachable" out to do this for more items :)
